Kickstarter-iOS/Views/Cells/PledgePaymentMethodsCell.swift

```diff
@@ -0,0 +1,83 @@
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class PledgePaymentMethodsCell: UITableViewCell, ValueCell {
+
+  // MARK: - Properties
+
+  private lazy var collectionView: UICollectionView = {
+    UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewLayout())
```

Comment:
Seeing as we know we're going to need a `UICollectionViewFlowLayout` here, should we instantiate one with a `horizontal` `scrollDirection`?